### PR TITLE
Move FxCop\System.Runtime.Analyzers project over

### PR DESCRIPTION
**Pull request by genlu**
- Copy System.Runtime.Analyzers from Roslyn repo.
- Update all projects to use shared targets/settings files, and references to correct NuGet packages.
- Add projects to the solution.

@srivatsn @tmeschter @mavasani

### CODE REVIEW

File: build/Targets/Analyzers.Versions.targets [hunk 1]
```diff
@@ -12,6 +12,12 @@
     <DesktopAnalyzersSemanticVersion>1.0.0</DesktopAnalyzersSemanticVersion>
     <DesktopAnalyzersPreReleaseVersion>beta1</DesktopAnalyzersPreReleaseVersion>
 
+    <SystemRuntimeAnalyzersSemanticVersion>1.0.2</SystemRuntimeAnalyzersSemanticVersion>
+    <SystemRuntimeAnalyzersPreReleaseVersion>beta1</SystemRuntimeAnalyzersPreReleaseVersion>
+    
+    <SystemRuntimeInteropServicesAnalyzersSemanticVersion>1.0.2</SystemRuntimeInteropServicesAnalyzersSemanticVersion>
+    <SystemRuntimeInteropServicesAnalyzersPreReleaseVersion>beta1</SystemRuntimeInteropServicesAnalyzersPreReleaseVersion>
+
```
Comment (genlu) [inline build/Targets/Analyzers.Versions.targets]: This will be used in my next PR, so let's leave it here for now.

**Comment by tmeschter** [comment]
@genlu Did you add a project to create a .nupkg containing this analyzer?

**Comment by genlu** [comment]
@tmeschter Not yet. Is the plan to have one NuGet package for each xxx.Analyzers project?

**Comment by tmeschter** [comment]
@genlu Correct. Long-term we envision the analyzers being distributed with the APIs they cover, rather than as a single, monolithic package.

**Comment by genlu** [comment]
@tmeschter OK, please wait until I add the nuget project to merge this. Another question: what about people using .NET framework (so the libraries they use don't come as nuget packages)? Will there be a single package created for that?

**Comment by tmeschter** [comment]
@genlu That's a valid point. End-users could still install the individual packages themselves, but it might make sense to create a meta-package that just pulls them all in. Could you open an issue about that?

**Comment by srivatsn** [comment]
The metapackage exists today - it is the Microsoft.FxCopAnalyzers package. It pulls in System.Runtime.Analyzers, System.InteropServices.Analyzers and Microsoft.AnalyzerPowerPack. As we add more API specific ones we should update that. We need to create an issue to move the nuget authoring of that package over from the roslyn repo.

**Comment by genlu** [comment]
@srivatsn I couldn't find Microsoft.FxCopAnalyzers in dotnet/roslyn anymore, do you know where it is located?

**Comment by srivatsn** [comment]
:+1:

**Pull request merged**